tests/integration/cmd_devfile_deploy_test.go

```diff
@@ -578,4 +580,35 @@ CMD ["npm", "start"]
 		})
 
 	})
+
+	Context("deploying devfile with long-running exec", func() {
+		BeforeEach(func() {
+			helper.CopyExampleDevFile(
+				filepath.Join("source", "devfiles", "nodejs", "devfile-deploy-exec-long.yaml"),
+				path.Join(commonVar.Context, "devfile.yaml"),
+				helper.DevfileMetadataNameSetter(cmpName))
+		})
+
```

Comment:
Can you add a similar test to what you did in `cmd_dev_test.go` to make sure that security context is not set if pod security is not enforced as restricted? https://github.com/redhat-developer/odo/blob/main/tests/integration/cmd_dev_test.go#L196